packages/firebase_remote_config/ios/Classes/FirebaseRemoteConfigPlugin.m

```diff
@@ -0,0 +1,166 @@
+#import "FirebaseRemoteConfigPlugin.h"
+
+#import <Firebase/Firebase.h>
+
+@interface FirebaseRemoteConfigPlugin ()
+@property(nonatomic, retain) FlutterMethodChannel *channel;
+@end
+
+@implementation FirebaseRemoteConfigPlugin
+
+static NSString *LAST_FETCH_TIME_KEY = @"LAST_FETCH_TIME";
+static NSString *LAST_FETCH_STATUS_KEY = @"LAST_FETCH_STATUS";
```

Comment:
I would use constants for all of the keys or none of them

Reply:
Done, removed key constants.